runtime/menu.vim

```diff
@@ -1287,4 +1287,27 @@ if has("gui_macvim")
   macm Help.MacVim\ Website			action=openWebsite:
 endif
 
+if has("touchbar")
```

Comment:
In the previous review there was a [comment](https://github.com/macvim-dev/macvim/pull/568#discussion_r148917612) about potentially having too many default Touch Bar buttons. I addressed this in the pull request descriptions, but I basically removed all the original ones and just added a full screen button that can toggle icon depending on what state it's in. I think most users will prefer a close to empty Touch Bar for customization so it's not worth having too many defaults (I just think having a couple that works well is a good thing to entice users to learn how to customize it).